### ISSUE TITLE
does not read the program for blind Nvda 2019.3

### ISSUE THREAD

**Issue by OlegDavidov** [flaviotordini/minitube#154]
For minitube version 2.9, the blind people program Nvda 2019.3 (https://www.nvaccess.org/download/) could read the youtube video list, and from the minitube 3.0 and above, the blind people program could not read the youtube video list in minitube .
(sorry for the translation)
